meetings/Agenda_2024-04-18.md

```diff
@@ -0,0 +1,20 @@
+## Meeting agenda April 18, 2024
+
+Topics to discuss:
+1. **Exec Check-in and debrief of year** (10-15 min)
+- Roses: What do you think went well this term? 
```

Comment:
- RGASC Help, CDRS was amaze for socials and wksps great workspace. Jess was amaze